mockingbird/processor/src/test/kotlin/com/anthonycr/mockingbird/processor/MockingbirdSymbolProcessorTest.kt

```diff
@@ -21,7 +22,10 @@ class MockingbirdSymbolProcessorTest {
         inheritClassPath = true
         verbose = false
         sources = list
-        symbolProcessorProviders = listOf(MockingbirdSymbolProcessorProvider())
+        symbolProcessorProviders += MockingbirdSymbolProcessorProvider()
+        configureKsp(useKsp2 = true) {
+            symbolProcessorProviders += MockingbirdSymbolProcessorProvider()
+        }
     }.compile()
 
     @Test
```

Comment:
The test sources need to have a package added in order to compile, it seems that the changes to the compile testing library won't create a file for a class without a package name. Tests work for me by adding a package name (just used `package com.anthonycr.test`) at the top of each test source.